charts/alertmanager/templates/alertmanager-networkpolicy.yaml

```diff
@@ -26,12 +26,22 @@ spec:
           tier: monitoring
           component: prometheus
           release: {{ .Release.Name }}
+    {{- if .Values.global.sidecar.enabled  }}
```

Comment:
this value of

Reply:
I guess that since the old one was static we don't need to add logic to allow more general use.

Reply:
okay daniel let me update those as well.

Reply:
sidecar name moved to authSidecar